fix graph dependencie does not have is_test set to False (#3311)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Calling **format** with the `-d` flag now removes test playbooks testing the deprecated content from conf.json.
* Fixed an issue where in some cases the **split** command did not remove pack version note from the script.

> * Fixed an issue where **validate** command did not catch dependencies of core packs.

Reviewer: ```suggestion
* Fixed an issue where **validate** would not properly detect dependencies of core packs.
```
